RHEL8: python check failure in lisav2

These logs repeat endlessly in RHEL8 RTM:

[Azure Functional WALA x] 05/07/2019 18:28:14 : [ERROR] Opened main channel
[Azure Functional WALA x] 05/07/2019 18:28:14 : [ERROR] Allocated pty (ospeed 38400bps, ispeed 38400bps)
[Azure Functional WALA x] 05/07/2019 18:28:14 : [ERROR] Started a shell/commandSent EOF messageServer sent command exit status 0Disconnected: All channels closed
[Azure Functional WALA x] 05/07/2019 18:28:14 : [ERROR] Linux machine returned exit code : 1
**[Azure Functional WALA x] 05/07/2019 18:28:14 : [ERROR] Failed to execute : which python || which python2 || which python3. Retrying...**

--- a/TestControllers/TestController.psm1
+++ b/TestControllers/TestController.psm1
@@ -341,8 +341,8 @@ Class TestController
 		}
 		if ($CurrentTestData.files -imatch ".py") {
 			$pythonPath = Run-LinuxCmd -Username $Username -password $Password -ip $VMData.PublicIP -Port $VMData.SSHPort `
-				-Command "which python || which python2 || which python3" -runAsSudo
-			if (($pythonPath -imatch "python2") -or ($pythonPath -imatch "python3")) {
+				-Command "which python || which python2 || which python3 || (which /usr/libexec/platform-python && ln -s /usr/libexec/platform-python /sbin/python)" -runAsSudo
+			if (!$pythonPath.Contains("platform-python") -and (($pythonPath -imatch "python2") -or ($pythonPath -imatch "python3"))) {
 				$pythonPathSymlink  = $pythonPath.Substring(0, $pythonPath.LastIndexOf("/") + 1)
 				$pythonPathSymlink  += "python"
 				Run-LinuxCmd -Username $Username -password $Password -ip $VMData.PublicIP -Port $VMData.SSHPort `

